images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -88,4 +75,15 @@ pacman.exe -Q | grep -v ^mingw-w64-
 
 Write-Host "`nMSYS2 installation completed"
 
+# Environment
+# add C:\msys64\mingw64\bin and C:\msys64\usr\bin to PATH
+# C:\msys64\mingw64\bin add after C:\Windows\System32 to not replace built-in tar.exe
+$regEnvKey = 'HKLM:\SYSTEM\CurrentControlSet\Control\Session Manager\Environment\'
+$pathValue = Get-ItemPropertyValue -Path $regEnvKey -Name 'Path'
+$pathValue += ";C:\msys64\mingw64\bin;C:\msys64\usr\bin"
+Set-ItemProperty -Path $regEnvKey -Name 'Path' -Value $pathValue
+
+# Rename python
+Rename-Item "C:\msys64\usr\bin\python.exe" -NewName "_python.exe"
```

Comment:
For example, this is an undesired effect of adding `C:\msys64\*\user\bin` to the PATH. Similar conflicts will exist for each tool that can be installed "natively" or through MSYS2.

Reply:
@al-cheb Why we need to rename python here?
Can we just put MSYS folders at the end of the PATH? I think it was working this way previously with Git

Reply:
Python fixed.